cime_config/testdefs/testlist_drv.xml

```diff
@@ -65,6 +65,16 @@
     </options>
   </test>
 
+  <test compset="X" grid="f19_g17" name="ERS_D_Ln9" testmods="drv/aoflux_ogrid">
+    <machines>
+      <machine name="derecho" compiler="intel" category="aux_cmeps"/>
```

Comment:
I felt like it would be good to have at least one test that still covers doing the aoflux calculation on the ogrid to ensure that that configuration continues to work. It felt right to add this to aux_cmeps since this is about testing cmeps code, but I'm not sure how often aux_cmeps is actually run. @jedwards4b do you have feelings on whether aux_cmeps is reasonable or if we should add this to prealpha or prebeta (instead or in addition)?

Also, I wasn't positive if an X compset actually covers the atm-ocn flux calculation, and so if this X compset test is sufficient for exercising aoflux_grid=ogrid. @jedwards4b or @mvertens do you know? If you aren't sure, I can try to confirm that we're covering relevant code with this test, or just change it to a B compset.

Reply:
I agree that retaining at least one ogrid case is a good idea.